### ISSUE TITLE
Input.json not being found on windows

### ISSUE THREAD

**Issue by rickj33** [open-policy-agent/vscode-opa#32]
I just installed the vscode-opa extension, version 0.6.0 and tried evaluating the rego. I get the following  error 
```
// Found 1 result in 0µs using no input file as input.
```
I have the rego, data.json and input.json in the workspace folder, I tried moving all of them to a sub folder but got the same result.

I think I traced the issue to the function getInputPath() in the extension.ts file. 
The parsed variable fsPath returns "C:\code\Discovery\Spikes\ColumnSecurity" however after the call to 
```
 let rootDir = opa.getDataDir(parsed);
```
the rootDir is set to "c:%5Ccode%5CDiscovery%5CSpikes%5CColumnSecurity"

and the return value of getInputPath is "c:%5Ccode%5CDiscovery%5CSpikes%5CColumnSecurityinput.json"

this value causes the existSync(path) to fail

**Comment by tsandall** [comment]
I'm looking into this. It's likely related to the recent improvement to support input.json in subdirs.

**Comment by tsandall** [comment]
This should be resolved in the latest version (v0.6.1). Closing for now.
